pangeo.md

```diff
@@ -1,6 +1,12 @@
 Pangeo: Earth Science
 =====================
 
+Copyright and License
+---------------------
+
+Copyright 2020 Ryan Abernathey. I license this work under a [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license.
```

Comment:
Thoughts on moving this to the bottom of the post? As https://github.com/dask/dask-stories/issues/3 showed, including a copyright notice is certainly important, but I wonder if it's the first this readers should see

Reply:
Fine with me.